libs/ui-web/src/lib/components/ui-web-input-host/ui-web-input-config.ts

```diff
@@ -1,13 +1,25 @@
 import { Injectable } from '@angular/core';
+import { Option, OptionType } from '@orchestrator/core';
 
-import { GenericFormAttributesConfig, InputType } from '../../types';
+import { FormAttributesConfig } from '../../form-attributes-config';
 
 @Injectable()
-export class UiWebInputConfig extends GenericFormAttributesConfig {
-  type?: InputType;
+export class UiWebInputConfig extends FormAttributesConfig {
+  @OptionType(String)
```

Comment:
Why it is `OptionType(String)` instead of `Option`?

Reply:
Patched